### PR TITLE
Added JSONP error handling for viz.json

### PR DESCRIPTION
**Pull request by redhog**
Resolves https://github.com/CartoDB/cartodb.js/issues/1153

### CODE REVIEW

**Comment by Cartofante** [comment]
Can one of the admins verify this patch?

File: src/core/loader.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ var Loader = cdb.vis.Loader = cdb.core.Loader = {
       script.type = 'text/javascript';
       script.src = src;
       script.async = true;
+      script.onerror = function (err) {
+        vizjson();
```
Comment (redhog) [inline src/core/loader.js]: vizjson() is defined in the global scope, and used by jsonp

**Comment by redhog** [comment]
@javisantana Could you take a look at this PR too?

**Comment by Cartofante** [comment]
Can one of the admins verify this patch?

**Comment by javisantana** [comment]
cc @alonsogarciapablo

**Comment by redhog** [comment]
Ping @alonsogarciapablo any chance of merging this fix?

**Comment by ivanmalagon** [comment]
Sorry Egil.

cartodb.js v3 is not under active development. Only major bugs are addressed.

Since there are no much movement in v3, feel free to use your own fork.

**Pull request closed without merging**